addons/account/views/account_move_views.xml

```diff
@@ -323,6 +323,7 @@
                     <field name="tax_ids" />
                     <field name="tax_line_id" string="Originator Tax"/>
                     <field name="reconcile_model_id"/>
+                    <field name="balance" string="Amount" filter_domain="['|', ('balance', '=', self), ('balance', '=', -self)]"/>
```

Comment:
Instead of using the balance in the filter domain and changing the self, look at debit and credit field :eyes: